openmdao/components/exec_comp.py

```diff
@@ -14,7 +14,7 @@
 from openmdao.core.explicitcomponent import ExplicitComponent
 
 # regex to check for variable names.
-VAR_RGX = re.compile('([_a-zA-Z]\w*[ ]*\(?)')
+VAR_RGX = re.compile('([.]*[_a-zA-Z]\w*[ ]*\(?)')
```

Comment:
was this a bug fix for something?

Reply:
yeah, there was something being treated as a variable that wasn't actually a variable.  I forget the details...